packages/nix/NIX.md

```diff
@@ -0,0 +1,31 @@
+# How to import
+
+To import with flakes use
+```nix
+inputs = {
+  polymc.url = "github:PolyMC/PolyMC";
+};
+
+...
+
+nixpkgs.overlays = [ inputs.polymc.overlay.${system} ]; ## Within configuration.nix
+environment.systemPackages = with pkgs; [ polymc ]; ##
+```
+
+To import without flakes use channels:
+
+```
+sudo -i nix-channel --add https://github.com/PolyMC/PolyMC/archive/master.tar.gz polymc
+sudo -i nix-channel --update polymc
```

Comment:
```suggestion
nix-channel --add https://github.com/PolyMC/PolyMC/archive/master.tar.gz polymc
nix-channel --update polymc
```
Principle of least privilege. Per-user channels work just fine, especially since we're using the per-user profile anyways.